### ISSUE TITLE
jotai breaking nextjs "next dev" command

### ISSUE THREAD

**Issue by kndwin** [pmndrs/jotai#667]
### Context
- Whilst trying to use jotai with the tailwindcss example from nextjs, I ran into an issue with after installing jotai, the `next dev` command no longer runs successfully

### Recreation
- `npx create-next-app -e with-tailwindcss my-project`
- `npm i jotai`
- `npm run dev`
```bash
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! @ dev: `next dev`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the @ dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/knd/.npm/_logs/2021-08-19T22_10_31_981Z-debug.log
```

### Troubleshotting attempts
- If I uninstall `jotai` with `npm uninstall jotai`, nextjs runs again.
- I tried removing `node_modules` with `rm -rf node_modules`, didn't work
- I also tried cleaning the npm cache with `npm clean cache --force`, also didn't work

### Other details
```bash
[ roomie main ] node -v
v14.17.0
[ roomie main ] npm -v
6.14.13
[ roomie main ] hostnamectl 
 Static hostname: donne
       Icon name: computer-laptop
         Chassis: laptop
      Machine ID: b1cf11dcbd9a4c828d1ce3fa6792dcb2
         Boot ID: c741ebf35c9a4c548b177437f644f6c2
Operating System: Arch Linux                      
          Kernel: Linux 5.13.7-arch1-1
    Architecture: x86-64
 Hardware Vendor: Dell Inc.
  Hardware Model: XPS 17 9700
[ roomie main ] cat package.json 
{
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start"
  },
  "dependencies": {
    "next": "latest",
    "react": "^17.0.2",
    "react-dom": "^17.0.2"
  },
  "devDependencies": {
    "autoprefixer": "^10.2.6",
    "postcss": "^8.3.5",
    "tailwindcss": "^2.2.4"
  }
}
[ roomie main ] cat /home/knd/.npm/_logs/2021-08-19T22_10_31_981Z-debug.log
0 info it worked if it ends with ok
1 verbose cli [ '/usr/local/bin/node', '/usr/local/bin/npm', 'run', 'dev' ]
2 info using npm@6.14.13
3 info using node@v14.17.0
4 verbose run-script [ 'predev', 'dev', 'postdev' ]
5 info lifecycle @~predev: @
6 info lifecycle @~dev: @
7 verbose lifecycle @~dev: unsafe-perm in lifecycle true
8 verbose lifecycle @~dev: PATH: /usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/knd/100sites/roomie/node_modules/.bin:/home/knd/.deta/bin:/home/knd/go:/home/knd/bin:/home/knd/.npm-packages/bin:/home/knd/.deno/bin:/opt/google-cloud-sdk/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/home/knd/.local/bin
9 verbose lifecycle @~dev: CWD: /home/knd/100sites/roomie
10 silly lifecycle @~dev: Args: [ '-c', 'next dev' ]
11 info lifecycle @~dev: Failed to exec dev script
12 verbose stack Error: @ dev: `next dev`
12 verbose stack spawn ENOENT
12 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:48:18)
12 verbose stack     at ChildProcess.emit (events.js:376:20)
12 verbose stack     at maybeClose (internal/child_process.js:1055:16)
12 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
13 verbose pkgid @
14 verbose cwd /home/knd/100sites/roomie
15 verbose Linux 5.13.7-arch1-1
16 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "dev"
17 verbose node v14.17.0
18 verbose npm  v6.14.13
19 error code ELIFECYCLE
20 error syscall spawn
21 error file sh
22 error errno ENOENT
23 error @ dev: `next dev`
23 error spawn ENOENT
24 error Failed at the @ dev script.
24 error This is probably not a problem with npm. There is likely additional logging output above.
25 verbose exit [ 1, true ]
```
Let me know if there's anything else I can provide or if this is just me.

**Comment by dai-shi** [comment]
Thanks for reporting.

While I don't understand the error messages, my hope is #657 would fix it.
Would you please try the 657 build?
https://ci.codesandbox.io/status/pmndrs/jotai/pr/657
Please find "Local Install Instructions".

**Comment by kndwin** [comment]
That did fix the issue! Wow thank you so much! Is there anything I can provide as post-mortem? If not, happy to close up this issue report :smile: 

```
[ roomie main ] npm i https://pkg.csb.dev/pmndrs/jotai/commit/d9e1f7ad/jotai
npm WARN @babel/plugin-syntax-jsx@7.14.5 requires a peer of @babel/core@^7.0.0-0 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

+ jotai@1.3.0
added 1 package from 1 contributor and audited 370 packages in 4.235s

64 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities



   ╭────────────────────────────────────────────────────────────────╮
   │                                                                │
   │      New major version of npm available! 6.14.13 → 7.21.0      │
   │   Changelog: https://github.com/npm/cli/releases/tag/v7.21.0   │
   │               Run npm install -g npm to update!                │
   │                                                                │
   ╰────────────────────────────────────────────────────────────────╯

[ roomie main ] npm run dev

> @ dev /home/knd/100sites/roomie
> next dev

ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - Using webpack 5. Reason: Enabled by default https://nextjs.org/docs/messages/webpack5

warn - You have enabled the JIT engine which is currently in preview.
warn - Preview features are not covered by semver, may introduce breaking changes, and can change at any time.
event - compiled successfully
wait  - compiling...
event - compiled successfully
event - build page: /next/dist/pages/_error
```

**Comment by dai-shi** [comment]
Thanks for confirming.
Please leave this open.
We'll let this be closed when #657 is merged.
